tensorflow/lite/delegates/gpu/BUILD

```diff
@@ -116,7 +116,7 @@ objc_library(
     alwayslink = 1,
 )
 
-# build -c opt --config android_arm64 --copt -Os --copt -DTFLITE_GPU_BINARY_RELEASE --copt --linkopt -s --strip always :libtensorflowlite_gpu_gl.so
```

Comment:
I think it should actually be the preceding `1--copt` which is removed.  That is, it should be

```
--copt -DTFLITE_GPU_BINARY_RELEASE --linkopt -s
```

Reply:
fixed in https://github.com/tensorflow/tensorflow/pull/45049/commits/54e6c38245185fe9e4eb6ad38b370d9b14562e97